bin/node-sass

```diff
@@ -229,63 +229,41 @@ function getOptions(args, options) {
  */
 
 function watch(options, emitter) {
-  var buildGraph = function(options) {
-    var graph;
-    var graphOptions = {
-      loadPaths: options.includePath,
-      extensions: ['scss', 'sass', 'css']
-    };
-
-    if (options.directory) {
-      graph = grapher.parseDir(options.directory, graphOptions);
-    } else {
-      graph = grapher.parseFile(options.src, graphOptions);
-    }
+  var handler = function(files) {
+    files.added.forEach(function(file) {
+      var watch = gaze.watched();
+      if (watch.indexOf(file) === -1) {
+        gaze.add(file);
+      }
+    });
 
-    return graph;
+    files.changed.forEach(function(file) {
+      if (path.basename(file)[0] !== '_') {
+        renderFile(file, options, emitter);
+      }
+    });
+
+    files.removed.forEach(function(file) {
```

Comment:
This could be

```
files.removed.forEach(gaze.remove)
```

but it's not likely the next watcher would have the same API